tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/DdwafObjectStruct.cs

```diff
@@ -25,7 +23,10 @@ internal struct DdwafObjectStruct
         // union
         // {
         [FieldOffset(16)]
-        public IntPtr RawHandle;
+        public IntPtr Array;
+
+        [FieldOffset(16)]
+        public IntPtr StringValue;
```

Comment:
Not, sure we need both of these fields if they're the same type, but not a big deal.

Reply:
removed stringvalue in 50c1359ab57b5cc2a9503358084816b10d950f65